.github/workflows/ci.yml

```diff
@@ -23,3 +23,30 @@ jobs:
     - name: Test
       run: |
         cmake --build build -t test
+  build-macos:
+    runs-on: macos-latest
```

Comment:
Could we think about Mac Apple silicon machines?

https://github.blog/changelog/2023-10-02-github-actions-apple-silicon-m1-macos-runners-are-now-available-in-public-beta/#:~:text=The%20GitHub%20Blog-,GitHub%20Actions%3A%20Apple%20silicon%20(M1)%20macOS%20runners%20are,now%20available%20in%20public%20beta!&text=Apple%20silicon%20(M1)%20hosted%20runners,or%20macos%2D13%2Dxlarge%20.